src/models/index.ts

```diff
@@ -1,17 +1,20 @@
 import {StateWithHistory} from 'redux-undo';
 
+import {Compass, DEFAULT_COMPASS} from './compass';
 import {Dataset, DEFAULT_DATASET} from './dataset';
 import {DEFAULT_SHELF_SPEC, Shelf} from './shelf';
 
 export * from './dataset';
 export * from './shelf';
+export * from './compass';
 
 /**
  * Application state.
  */
 export interface StateBase {
   dataset: Dataset;
   shelf: Shelf;
+  compass: Compass;
```

Comment:
Let's make this `results: ResultIndex` as it is essentially a index of results from CompassQL Query